src/main/resources/reference.conf

```diff
@@ -25,4 +25,35 @@ spray.can {
     request-timeout = 40s
     connecting-timeout = 40s
   }
-}
\ No newline at end of file
+}
```

Comment:
This does a better job than I could: https://github.com/typesafehub/config#standard-behavior

It's not a big deal, but IMO strictly speaking we should be using application.